docs/site/soap-calculator-tutorial-add-controller.md

```diff
@@ -106,14 +106,14 @@ service are available from here due to the dependency injection available
 through out **LB4**.
 
 ```ts
- async divide(intA: number,intB: number): Promise<DivideResult> {
+ async divide(arg0: number,arg1: number): Promise<DivideResponse> {
```

Comment:
I personally find `intA` and `intB` a bit more descriptive than `arg0` and `arg1`.

Reply:
I haven't found a way to customize such argument names.

Reply:
Ok, I found a way :-)